scripts/zones/Throne_Room/bcnms/shadow_lord_battle.lua

```diff
@@ -51,7 +51,7 @@ end;
 
 function onEventFinish(player,csid,option)
     -- print("bc finish csid "..csid.." and option "..option);
-    if (csid == 7d01) then
+    if (csid == 0x07d01) then
```

Comment:
This should have been fixed when we merges master back to bcnmcrap last. its means something wasn't merged right (or at all, I was sure we did a back merge already..) which has me concerned some other things are missing now.

Reply:
I think last time demo merged in master, he just picked bcnmcrap's scripts for all bcnms, so this problem probably doesn't extend past bcnm scripts.

Reply:
I had worked on merging them correctly and pr'd it to bcnm crap though... no reason it would have been wrecked after. That behavior will rebreak a lot of stuff in the future when we finally merge bcnm crap to master which was why I did that for you :-(

Reply:
I'll compare shiz to try and spot the breaks iver the weekend and pr to bcnm crap again